frontend/pages/hosts/details/cards/About/About.tsx

```diff
@@ -137,6 +137,18 @@ const About = ({
     );
   };
 
+  const renderBattery = () => {
+    if (typeof aboutData.batteries !== "object") {
+      return null;
+    }
+    return (
+      <div className="info-grid__block">
+        <span className="info-grid__header">Battery</span>
+        <span className="info-grid__data">{aboutData.batteries[0].health}</span>
```

Comment:
Do you want to use any optional chaining here? (e.g., `aboutData?.batteries?.[0]?.health` but you probably don't need all of them assuming you are confident the value will exist)

Reply:
Doesn't hurt! Updated. 

The `aboutData` object is set to `any` rather than having a defined interface. That's resulting in quite a few other places in this file where there aren't checks, which of course makes the UI brittle. I'll write up / prioritize an issue to address this throughout the app.

Reply:
> The aboutData object is set to any rather than having a defined interface. 
Exactly what I was afraid of!